src/shared/components/common/registration-application.tsx

```diff
@@ -69,8 +69,9 @@ export class RegistrationApplication extends Component<
           />
         </div>
         <div>
-          {I18NextService.i18n.t("created")}:{" "}
-          <MomentTime showAgo published={ra.published_at} />
+          {I18NextService.i18n.t("created_at", {
+            time: formatDate(ra.published_at),
+          })}
```

Comment:
This should still use the MomentTime IMO. The hover / click and hold will still show the real date.